src/Polysemy/Internal.hs

```diff
@@ -394,14 +399,16 @@ raise_ = mapMembership raiseMembership
 class Raise (r :: EffectRow) (r' :: EffectRow) where
   raiseMembership :: ElemOf e r -> ElemOf e r'
 
-instance {-# overlapping #-} Raise r r where
-  raiseMembership = id
+instance {-# INCOHERENT #-} Raise r r where
+  raiseMembership = idMembership
   {-# INLINE raiseMembership #-}
 
-instance (r' ~ (_0 ': r''), Subsume r r'') => Raise r r' where
-  raiseMembership = There . subsumeMembership
+instance {-# OVERLAPPING #-} Raise (e ': r) (e ': r) where
+  raiseMembership = idMembership
   {-# INLINE raiseMembership #-}
 
+instance Raise r r' => Raise r (_0 ': r') where
+  raiseMembership = There . raiseMembership
```

Comment:
These changes don't change the semantics of `Raise` any, just gives them better error messages (error messages under the old implementation were... _frightening_.)

Reply:
Oh, I moved `Raise` back to not using `Subsume`, instead `Subsume` uses a copy of `Raise` (... which should also receive these changes, oops).